.github/workflows/te-review.yml

```diff
@@ -52,20 +51,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-#        Disable te-empl and te-yout due to YJDH-370
```

Comment:
Why not just remove the lines instead of commenting?

Reply:
This is removing, they were previously commented so I just removed the commented out lines as unnecessary now.

Reply:
My eyes have something wrong today 😎